src/components/calculator.jsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/button-has-type */
```

Comment:
- [ ] For the sake of following project requirements, please make this file a `.js` file not `.jsx` as there is no difference it makes here. :relaxed: